### PR TITLE
SASS-10500 - Create backend endpoints to store 'journey-answers' for CIS

### PR DESCRIPTION
**Pull request by stephenheath101**
### Description
Please include a brief summary of the change you've made and the issue that it fixes.

Add a link to the relevant story in Jira
[SASS-CHANGE-ME](ADD URL HERE)

##### Before Reviewing
- [ ]  Have you pulled the branch down?
- [ ]  Have you assigned yourself to the PR?
- [ ]  Have you moved the task to “in review” on JIRA?
- [ ]  Have you checked to ensure all dependencies are up to date?
- [ ]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
- [ ]  Have you run the tests?
- [ ]  Have you run the journey tests?
- [ ]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [ ]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [ ]  Have you run the tests?
- [ ]  Have you run the journey tests? (where applicable)
- [ ]  Have you addressed warnings where appropriate?
- [ ]  Have you rebased against the current version of main?
- [ ]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [ ]  Have you checked the PR Builder passes?

### CODE REVIEW

**Review by BetaDraconis — APPROVED** [review]

File: app/config/Module.scala [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
```
Comment (garytate123) [inline app/config/Module.scala:2]: Copyright year should be 2024

File: app/models/mongo/JourneyAnswers.scala [hunk 2]
```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
```
Comment (garytate123) [inline app/models/mongo/JourneyAnswers.scala:2]: Copyright year should be 2024

File: conf/app.routes [hunk 3]
```diff
@@ -6,5 +6,11 @@ DELETE  /income-tax/nino/:nino/sources/:submissionId        controllers.DeleteCI
 
 POST    /income-tax/nino/:nino/sources                      controllers.CreateUpdateCisDeductionsController.postCISDeductions(nino: String, taxYear: Int)
 
+# Journey answers endpoints
+GET         /income-tax/journey-answers/:journeyKey/:taxYear                   controllers.JourneyAnswersController.get(journeyKey: String, taxYear: TaxYear)
+POST        /income-tax/journey-answers                                        controllers.JourneyAnswersController.set
+DELETE	    /income-tax/journey-answers/:journeyKey/:taxYear                   controllers.JourneyAnswersController.clear(journeyKey: String, taxYear: TaxYear)
+POST        /income-tax/journey-answers/keep-alive/:journeyKey/:taxYear        controllers.JourneyAnswersController.keepAlive(journeyKey: String, taxYear: TaxYear)
+
 GET     /:taxYear/tasks/:nino                               controllers.CommonTaskListController.getCommonTaskList(taxYear: Int, nino: String)
```
Comment (garytate123) [inline conf/app.routes:15]: Same as above, with aligning the controllers up

File: it/test/support/IntegrationTest.scala [hunk 4]
```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
```
Comment (garytate123) [inline it/test/support/IntegrationTest.scala:2]: Copyright year should be 2024

**Review by garytate123 — APPROVED** [review]

**Review by basotia — APPROVED** [review]

**Pull request merged**